requirements/edx/github.txt

```diff
@@ -8,6 +8,7 @@
 # Python libraries to install directly from github
 
 # Third-party:
+git+https://github.com/edx/factory_boy@2d5b4df960c5b1076aae5ead6b9ddbe3aa50bcd8
```

Comment:
I won't force the matter, since we don't really have a release strategy for forked libraries; but, it would be nice to use a release tag for this as well. This is purely optional, however.